modules/zivid/experimental/point_cloud_export/file_format.py

```diff
@@ -0,0 +1,185 @@
+import _zivid
+
+
+class ColorSpace:
+    """Color space for saving point cloud."""
+
+    linear_rgb = "linear_rgb"
+    srgb = "srgb"
+
+    @staticmethod
+    def valid_values():
+        """Get valid values for color space."""
+        return [ColorSpace.linear_rgb, ColorSpace.srgb]
+
+    @classmethod
+    def _to_internal(cls, value):
+        if value == ColorSpace.linear_rgb:
+            return _zivid.point_cloud_export.ColorSpace.linear_rgb
+        if value == ColorSpace.srgb:
+            return _zivid.point_cloud_export.ColorSpace.srgb
+        raise ValueError(
+            "Invalid color space '{}'. Valid color spaces are: {}".format(
+                value, cls.valid_values()
+            )
+        )
+
+
+class ZDF:
+    """Specification for saving frame in ZDF (*.zdf) format."""
+
+    def __init__(self, file_name):
+        """Create a ZDF file format specification with file name.
+
+        Args:
+            file_name: File name.
+        """
+        if not isinstance(file_name, str):
+            raise TypeError(
+                "Unsupported type for argument file_name. Got {}, expected {}".format(
+                    type(file_name), str
+                )
+            )
+        self.__impl = _zivid.point_cloud_export.file_format.ZDF(file_name)
+
+    def __str__(self):
+        return str(self.__impl)
+
+
+class PLY:
+    """Specification for saving frame in PLY (*.ply) format.
+
+    PLY is a file format developed at Stanford. To learn more about the PLY file format,
+    see https://paulbourke.net/dataformats/ply/.
+    """
+
+    class Layout:
+        """Layout for saving point cloud."""
+
+        ordered = "ordered"
+        unordered = "unordered"
+
+        @staticmethod
+        def valid_values():
+            """Get valid values for layout."""
+            return [PLY.Layout.ordered, PLY.Layout.unordered]
+
+        @classmethod
+        def _to_internal(cls, value):
+            if value == PLY.Layout.ordered:
+                return _zivid.point_cloud_export.file_format.PLY.Layout.ordered
+            if value == PLY.Layout.unordered:
+                return _zivid.point_cloud_export.file_format.PLY.Layout.unordered
+            raise ValueError(
+                "Invalid layout '{}'. Valid layouts are: {}".format(
+                    value, cls.valid_values()
+                )
+            )
+
+    def __init__(
+        self, file_name, layout=Layout.ordered, color_space=ColorSpace.linear_rgb
```

Comment:
default in sdk is updated to `ColorSpace.srgb` for all file formats.

Reply:
It would be best if we could just inherit the default behavior from the SDK side?

Reply:
We could do that. But then should we also mention the default in the comment? At least if we set the default in python also we can see that the documentation is correct.